odl/solvers/util/callback.py

```diff
@@ -385,39 +401,63 @@ class CallbackPrintTiming(SolverCallback):
 
     """Callback for printing the time elapsed since the previous iteration."""
 
-    def __init__(self):
-        """Initialize a new instance."""
+    def __init__(self, fmt='Time elapsed = {:<5.03f} s', step=1):
+        """Initialize a new instance.
+
+        Parameters
+        ----------
+        fmt : string, optional
+            Formating that should be applied.
```

Comment:
While correct, "formating" with one "t" is less common.

Also print the usage pattern, like
```
Will be used as ::

    print(fmt.format(t_current - t_previous))
```